emojis.json

```diff
@@ -2895,7 +2895,7 @@
     "category": "travel_and_places"
   },
   "hotel": {
-    "keywords": ["building", "whotel", "accomodation", "checkin"],
+    "keywords": ["building", "hotel", "accomodation", "checkin"],
```

Comment:
The test is catching that the emoji name doesn't need to be the keyword, just removing "hotel" would do.